Practice3/Validator.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+
+namespace Practice3
+{
+    public static class Validator
```

Comment:
the \n is missing somewhere

![image](https://user-images.githubusercontent.com/9382934/113509299-7b6a6700-955d-11eb-8eba-b9d8325db31a.png)